### PR TITLE
updating the TOC and add reference to Adding user workload metrics

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-3500

The author of the Jira issue requested for us to add a statement to instruct the user to take a look at the _Adding user workload metrics_ section.

### CODE REVIEW

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -8,15 +8,16 @@ Collect logs about new information that is created for observability resources w
...
 ** <<adding-user-workload-metrics,Adding user workload metrics>>
-** <<updating-the-multiclusterobservability-cr,Updating the multiclusterobservability CR>>
+** <<exporting-metrics-to-external-endpoints,Exporting metrics to external endpoints>>
+** <<removing-default-metrics,Removing default metrics>>
+- <<creating-the-kubernetes-secret-for-external-endpoint,Creating the Kubernetes secret for an external endpoint>>
+** <<updating-the-multiclusterobservability-cr,Updating the multiclusterobservability custom resource>>
 ** <<viewing-the-status-of-metrics-export,Viewing the status of metric export>>
 - <<adding-advanced-config,Adding _advanced_ configuration>>
-- <<updating-replicas,Updating the multiclusterobservability CR replicas from the console>>
+- <<updating-replicas,Updating the multiclusterobservability custom resource replicas from the console>>
```
Comment (oafischer) [inline observability/customize_observability.adoc]: Since `multiclusterobservability` is monospace further down, should we italicize it here?

**Review by oafischer — APPROVED** [review]
One comment for consideration

File: observability/customize_observability.adoc (hunk 1 above)
Comment (dockerymick) [reply observability/customize_observability.adoc]: Yes! Great catch Oliver

**Comment by dockerymick** [comment]
Closing this PR, opened a new to confirm all Linting problems were resolved.

**Comment by dockerymick** [comment]
https://github.com/stolostron/rhacm-docs/pull/4791

**Pull request closed without merging**